project/dexter-monitor/app.js

```diff
@@ -164,6 +164,7 @@ function setWebApis(){
 
     app.get('/api/v2/project-list', project.getProjectList);
     app.get('/api/v2/snapshot-summary', project.getSnapshotSummary);
+	app.post('/api/v2/create-project',project.createProject);
```

Comment:
Please use noun only as a REST API name.
I think that `app.post('/api/v2/project', project.createProject);` is enough.